.golangci.yml

```diff
@@ -22,21 +22,21 @@ linters:
     - bodyclose
     - deadcode
     - decorder
-    - depguard
+    # - depguard
```

Comment:
I'm fine with removing `depguard` and `goconst`, they don't seem to be particularly useful to our use case. Can you add a `TODO` to the `gosec` one please? I'll create a PR to fix that today.

Reply:
Updated

Reply:
Can you please remove the two lines instead of commenting them out? The ones without the TODO that is.

Reply:
Updated.